iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -161,13 +168,12 @@ class ConversationListViewController: FullScreenViewController,
         self.didCenterOnCell = cell
 
         // If there's a centered cell, update the layout
-        if let indexPath = self.collectionView.centerIndexPath() {
-
-            if let conversation = self.conversationListController.conversations[safe: indexPath.item] {
-                self.messageInputAccessoryView.conversation = conversation
-                self.conversationHeader.configure(with: conversation)
-            }
+        if let currentConversation = self.currentConversation {
+            ConversationsManager.shared.activeConversations.removeAll()
+            ConversationsManager.shared.activeConversations.append(currentConversation)
```

Comment:
Updating the active conversation when a new conversation is centered.

Reply:
nice